core/tests/org.openjdk.jmc.jdp.test/src/test/java/org/openjdk/jmc/jdp/client/TestToolkit.java

```diff
@@ -62,6 +69,7 @@
 			JDPClientTest.LOGGER.log(Level.SEVERE, "Could not create test multicast address!", e);
 		}
 		TEST_MULTICAST_ADDRESS = tmp;
+		BROADCASTING_SUPPORTED = isBroadcastingSupported();
```

Comment:
Hm, I'm wondering whether this broadcasting check should be done in the static initialization of the class... maybe not. Rather do it lazily when calling isBroadcastingSupported()? There are a few tests which don't need this...

Reply:
The class is only used in this package and half of the test classes in this package need to functionality for their test.
There are 4 test classes overall.